modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -30,101 +30,39 @@ Intel processors that support the Advanced Vector Extensions 2 (AVX2) instructio
-Intel processors that support the Advanced Vector Extensions 2 (AVX2) instruction set.
-To verify whether your device supports the AVX2 instructions set, https://www.intel.com/content/www/us/en/support/articles/000090473/processors/intel-core-processors.html[follow these instructions.] 
---
-
-A beta version of Vector Search is now available on Couchbase Lite for Java Desktop with updates to the iOS (Swift, Objective-C) and Android platforms.
-
-Read more at: 
-
 * xref:java:vector-search.adoc[Vector Search - Java Desktop]
 ** xref:java:gs-install.adoc[Installation Instructions]
 ** xref:java:working-with-vector-search.adoc[Use Vector Search]
 
-* xref:swift:vector-search.adoc[Vector Search - Swift]
-** xref:swift:gs-install.adoc[Installation Instructions]
-** xref:swift:working-with-vector-search.adoc[Use Vector Search]
-
 * xref:objc:vector-search.adoc[Vector Search - Objective-C]
 ** xref:objc:gs-install.adoc[Installation Instructions]
 ** xref:objc:working-with-vector-search.adoc[Use Vector Search]
 
-* xref:android:vector-search.adoc[Vector Search - Android]
-** xref:android:gs-install.adoc[Installation Instructions]
-** xref:android:working-with-vector-search.adoc[Use Vector Search]
-
-== Release 3.2.0 - Beta 1 - iOS and Android (March 2024)
-
-=== New Features
-
-==== Vector Search
-
-IMPORTANT: Vector Search is only available for 64-bit architectures.
-
-A beta version of Vector Search is now available on Couchbase Lite for iOS (Swift, Objective-C) and Android.
-Vector search is a sophisticated data retrieval technique that focuses on matching the contextual meanings of search queries and data entries, rather than simple text matching.
-Vectors are represented by arrays of numbers known as embeddings, which are generated by Large Language Models (LLMs) to represent objects such as text, images, and audio.
-You can use Vector Search to efficiently find similar items or content based on the similarity of their vector representations. 
-This is useful for reducing the cost per query, performing semantic or similarity search, providing recommendations among others.
-
-Read more at: 
-
 * xref:swift:vector-search.adoc[Vector Search - Swift]
 ** xref:swift:gs-install.adoc[Installation Instructions]
 ** xref:swift:working-with-vector-search.adoc[Use Vector Search]
 
-* xref:objc:vector-search.adoc[Vector Search - Objective-C]
-** xref:objc:gs-install.adoc[Installation Instructions]
-** xref:objc:working-with-vector-search.adoc[Use Vector Search]
-
-* xref:android:vector-search.adoc[Vector Search - Android]
-** xref:android:gs-install.adoc[Installation Instructions]
-** xref:android:working-with-vector-search.adoc[Use Vector Search]
-
-* xref:java:vector-search.adoc[Vector Search - Java Desktop]
-** xref:java:gs-install.adoc[Installation Instructions]
-** xref:java:working-with-vector-search.adoc[Use Vector Search]
-
 ==== Extended Date/Time Functionality
 
 Six new DateTime functions have been added to Couchbase Lite N1Q1:
```

Comment:
I know it's not part of this change, but should this be `{sqlpp}` or at least `N1QL`?

Reply:
Changed to N1QL